app/controllers/application_controller.rb

```diff
@@ -1,6 +1,16 @@
 class ApplicationController < ActionController::Base
-  # strong params fo rdevise is on application controller 
-  before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
this same code is also on line 3

Reply:
weird maybe i didnt save it properly

Reply:
wait nevermind it should be fine, not sure what you're referring to